doc/stdenv.xml

```diff
@@ -747,7 +747,8 @@ passthru.updateScript = [ ../../update.sh pname "--requested-release=unstable" ]
    latter is convenient from a build script. However, typically one only wants
    to <emphasis>add</emphasis> some commands to a phase, e.g. by defining
    <literal>postInstall</literal> or <literal>preFixup</literal>, as skipping
-   some of the default actions may have unexpected consequences.
+   some of the default actions may have unexpected consequences. The default
+   script for each phase is defined in the package stdenv/setup.
```

Comment:
I'm not sure if that's really clear.  Perhaps "file `pkgs/stdenv/generic/setup.sh`"?